pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>common</artifactId>
-        <version>5.4.0-SNAPSHOT</version>
+        <version>5.1.3</version>
```

Comment:
Why do we change that? I'd rather like to keep the version it was forked off

Reply:
Because then we have to build the snapshots locally - they arent available on their maven repository.